FSW/src/headers/config.hpp

```diff
@@ -6,7 +6,7 @@
  *  sampling rates, etc) that we will want to be able to change
  *  from a single location and include in multiple modules
  * 
- *  declare all objects as static to avoid the error 
+ *  declare all objects as extern to avoid the error 
  *      "multiple definitions of <object>" during compilation

```

Comment:
might make sense to explain the relationship between `config.hpp` and `externalConfig.hpp` here? something along the lines of 

`config.hpp` contains all constants that may be useful to change before running the FSW. `externalConfig.hpp` is used to instantiate class objects using `extern` so that they can be modified from any other files in the FSW.

Reply:
changed to:

"declare all non-constant objects as 'extern' and define them in externalDefinitions.cpp to create a single instance shared across all files. Objects not declared as 'extern' will not be linked between files."

I also renamed `externalConfig` to `externalDefinitions` like you suggested, since none of the init values are actually defined there.